elementary/monitor/dbt_project/macros/validate_source_freshness_invocation.sql

```diff
@@ -0,0 +1,19 @@
+{% macro validate_source_freshness_invocation(invocation_id, days_back=14) %}
```

Comment:
Care to rename this macro to something that indicates that it checks whether the source freshness was uploaded or not?

Reply:
renaming the macro to `can_upload_source_freshness`, and the internal variables as well